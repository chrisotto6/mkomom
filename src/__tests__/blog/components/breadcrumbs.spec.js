import React from 'react'
import { create } from 'react-test-renderer'
import { StaticQuery } from 'gatsby'
import Breadcrumbs from '../../../blog/components/breadcrumbs'

describe('Breadcrumbs', () => {
  it('renders correctly', () => {
    const tree = create(
      <Breadcrumbs lastPath="test" lastName="test" currentPage="test page" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
