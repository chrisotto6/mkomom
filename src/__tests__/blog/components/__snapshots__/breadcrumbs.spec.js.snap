// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders correctly 1`] = `
<div
  className="w-100 f6 pv3 flex items-center tracked ttu sans-serif justify-around mw5"
>
  <a
    className="dark-gray"
    href="/"
  >
    Home
  </a>
  <span>
     &gt; 
  </span>
  <a
    className="dark-gray"
    href="test"
  >
    test
  </a>
  <span>
     &gt; 
  </span>
  <span
    className="mid-gray"
  >
    test page
  </span>
</div>
`;
